File: Python/problem029.py
#! /usr/bin/env python3
'''
Problem 29 - Project Euler
http://projecteuler.net/index.php?section=problems&id=029
'''
if __name__ == '__main__':
    ans = len(set([a ** b for a in range(2, 101) for b in range(2, 101)]))
    print(ans)

File: Clojure/problem029.clj
#! /usr/bin/env clojure
; Problem 29 - Project Euler
; http://projecteuler.net/index.php?section=problems&id=029

(require '[mathtools :refer [power]])
(-> (for [a (range 2 (inc 100))
          b (range 2 (inc 100))]
      (power a b))
    (distinct)
    (count)
    (println))
